src/components/layout/footer.module.scss

```diff
@@ -1,99 +1,143 @@
     }
   }
-  .footer-list {
+}
+
+.footer {
+  @include padding(8, bottom);
+  @include padding(32, top);
+
+  img {
+    width: 32px;
+  }
+
+  .project-logo {
+    width: toRem(176);
+    margin: 0;
+    @include margin(16, right);
+  }
+}
+
+.main {
+  @include row();
+  align-items: center;
+
+  ul {
     list-style-type: none;
-    padding: 0;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: right;
-    @media (min-width: $viewport-md) {
-      margin: 0;
+    @include padding(0, left);
+  }
+
+  .ctp-logo {
+    @include col(3 3 3);
+
+    @media (max-width: $viewport-lg) {
+      order: 1;
     }
+  }
 
-    @media (min-width: $viewport-sm) {
-      li {
-        display: inline-block;
-        margin: spacer(4) 0 spacer(4) spacer(24);
-        margin-bottom: 0;
-        &:last-of-type {
-          margin-right: 0;
-        }
-        &:first-of-type {
-          margin-left: 0;
-        }
+  .links {
+    @include col(4 6 7);
+    ul {
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-rows: repeat(2, 1fr);
+      grid-auto-flow: column;
+      @media (max-width: $viewport-lg) and (min-width: $viewport-md) {
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+      }
+      @media (max-width: $viewport-md) {
+        grid-template-columns: 1fr;
+        grid-template-rows: repeat(6, 1fr);
       }
     }
 
-    @media (max-width: $viewport-ms) and (min-width: $viewport-sm) {
-      li:first-of-type {
-        margin-left: 1.5rem;
-      }
+    @media (max-width: $viewport-lg) {
+      order: 3;
     }
+  }
 
-    a {
-      color: white;
-      font-weight: bold;
-      text-decoration: none;
-      &:hover {
-        @include link-hover-dark-bg;
-      }
+  .social {
+    @include col(1 3 2);
+    @include margin(0, (top, bottom));
+
+    display: flex;
+    justify-content: right;
+
+    @media (max-width: $viewport-lg) {
+      order: 2;
     }
-    .back-to-top {
-      display: block;
-      margin-top: spacer(32);
-      font-weight: bold;
-      @media (min-width: $viewport-sm) {
-        display: inline;
-        margin-top: 0;
+
+    li {
+      @include margin(16, left);
+      &:first-of-type {
+        @include margin(0, left);
       }
     }
   }
 }
 
+hr.divider {
+  background-color: $color-plum-500;
+}
+
 .copyright {
-  padding: 1.5rem 2rem 0.5rem;
-  text-align: center;
-  font-size: 1rem;
+  @include padding(32, bottom);
+  @include padding(8, top);
 
-  a {
-    color: white;
-    &:hover {
-      @include link-hover-dark-bg;
+  @include row();
+
+  justify-content: space-between;
+  align-items: center;
+
+  span {
+    @include col(3 5 10);
+    a {
+      font-weight: normal;
+      display: inline;
+      text-decoration: underline;
     }
-    &:focus {
-      @include link-focus-dark-bg;
+  }
+
+  .back-to-top {
+    @include col(1 1 2);
+
+    display: flex;
+    align-items: center;
+    justify-content: right;
+
```

Comment:
Let's also have the back-to-top link float to the right below lg breakpoints so it's flush.

Reply:
I think this is already the case. Here's a screenshot:

![image](https://user-images.githubusercontent.com/18607205/85186270-50748800-b255-11ea-8d5e-0aef30428068.png)

Reply:
Fixed in 23cf28d0b3dba16f8f52f0994cdb412d44c1f7f0 -- see the commit message for details